### ISSUE TITLE
Incorrect behaviour for @var annotation of property

### ISSUE THREAD

**Issue by racinmat** [slevomat/coding-standard#183]
I have entity with properties and  coding standard complains about not having @var annotation there, although my class looks like this:
```php
<?php declare(strict_types=1);

namespace AppBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * TournamentMatch
 *
 * @ORM\Table(name="tournament_match")
 * @ORM\Entity()
 * @ORM\HasLifecycleCallbacks()
 */
class TournamentMatch extends BaseEntity
{
    /**
     * @var int
     * @ORM\Column(name="match_index", type="integer", nullable=false)
     */
    private $matchIndex;

    /**
     * @var int
     * @ORM\Column(name="score0", type="integer", nullable=true)
     */
    private $score0;

    /**
     * @var int
     * @ORM\Column(name="score1", type="integer", nullable=true)
     */
    private $score1;

    /**
     * @var \AppBundle\Entity\Tournament
     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\Tournament")
     */
    private $tournament;

    /**
     * @var \AppBundle\Entity\TournamentRegistration
     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\TournamentRegistration")
     */
    private $registration0;

    /**
     * @var \AppBundle\Entity\TournamentRegistration
     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\TournamentRegistration")
     */
    private $registration1;
    
    //...
}

```
and coding standard output is:

 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Line   src\AppBundle\Entity\TournamentMatch.php
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  1      Property \AppBundle\Entity\TournamentMatch::$matchIndex does not have @var annotation.
         (SlevomatCodingStandard\Sniffs\TypeHints\TypeHintDeclarationSniff)
  1      Property \AppBundle\Entity\TournamentMatch::$score0 does not have @var annotation.
         (SlevomatCodingStandard\Sniffs\TypeHints\TypeHintDeclarationSniff)
  1      Property \AppBundle\Entity\TournamentMatch::$score1 does not have @var annotation.
         (SlevomatCodingStandard\Sniffs\TypeHints\TypeHintDeclarationSniff)
  1      Property \AppBundle\Entity\TournamentMatch::$tournament does not have @var annotation.
         (SlevomatCodingStandard\Sniffs\TypeHints\TypeHintDeclarationSniff)
  1      Property \AppBundle\Entity\TournamentMatch::$registration0 does not have @var annotation.
         (SlevomatCodingStandard\Sniffs\TypeHints\TypeHintDeclarationSniff)
  1      Property \AppBundle\Entity\TournamentMatch::$registration1 does not have @var annotation.
         (SlevomatCodingStandard\Sniffs\TypeHints\TypeHintDeclarationSniff)
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Line   src\AppBundle\Form\AddScoreType.php
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  12     Method \AppBundle\Form\AddScoreType::buildForm() does not have @param annotation for its traversable parameter $options.
         (SlevomatCodingStandard\Sniffs\TypeHints\TypeHintDeclarationSniff)
  23     Method \AppBundle\Form\AddScoreType::getBlockPrefix() does not have return type hint nor @return annotation for its return value.
         (SlevomatCodingStandard\Sniffs\TypeHints\TypeHintDeclarationSniff)
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Line   src\AppBundle\Repository\PlayerRepository.php
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  1      Method \AppBundle\Repository\PlayerRepository::getInvitablePlayerUsernames() does not have @return annotation for its traversable return value.
         (SlevomatCodingStandard\Sniffs\TypeHints\TypeHintDeclarationSniff)
  1      Method \AppBundle\Repository\PlayerRepository::getInvitablePlayerUsernames() does not need documentation comment.
         (SlevomatCodingStandard\Sniffs\TypeHints\TypeHintDeclarationSniff)
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

**Comment by kukulich** [comment]
@racinmat Could you please post you ruleset.xml? What version of PHPCS do you use? What version of our standard do you use? I'm currently not able to reproduce it.

**Comment by kukulich** [comment]
Closing because of no response.

**Comment by racinmat** [comment]
I tried to reproduce this my bug, but with no luck, so there was probably some magic. It works now.
